data/src/main/java/io/gnosis/data/repositories/EnsRepository.kt

```diff
@@ -2,7 +2,10 @@ package io.gnosis.data.repositories
 
 import io.gnosis.contracts.BuildConfig
 import pm.gnosis.crypto.utils.Sha3Utils
-import pm.gnosis.ethereum.*
+import pm.gnosis.ethereum.Block
+import pm.gnosis.ethereum.EthCall
+import pm.gnosis.ethereum.EthRequest
+import pm.gnosis.ethereum.EthereumRepository
```

Comment:
Are these unused imports or the just got unfolded by the IDE?

Reply:
These are not unused imports.